tsconfig.json

```diff
@@ -1,30 +1,6 @@
 {
-  "compilerOptions": {
```

Comment:
I'd rather make this config for base config either cjs or es2015 config and get rid one of them. don't feel we need unusued one.